presto-hive/src/test/java/io/prestosql/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -343,6 +346,45 @@ public void testNoHangIfPartitionIsOffline()
         assertThrows(RuntimeException.class, hiveSplitSource::isFinished);
     }
 
+    @Test(timeOut = 6_000)
```

Comment:
increase timeout to 30s to reduce chance for flakyness

Reply:
done